custom_types/ledgerhq__hw-transport-node-hid/index.d.ts

```diff
@@ -1 +1,38 @@
-declare module "@ledgerhq/hw-transport-node-hid";
+declare module "@ledgerhq/hw-transport-node-hid" {
+  import { Device, HID } from "node-hid";
+
+  /**
+   * "A Descriptor is a parametric type that is up to be determined for the
+   * implementation. it can be for instance an ID, an file path, a URL,..."
```

Comment:
`a file path`

Reply:
haha, nice catch. You need to add an upstream PR to fix the source of that quote ;)

Reply:
fixed